docs/containers-registries.conf.5.md

```diff
@@ -108,16 +108,25 @@ Each TOML table in the `mirror` array can contain the following fields, with the
 as if specified in the `[[registry]]` TOML table directly:
 - `location`
 - `insecure`
+- `digest-only`
 
 `mirror-by-digest-only`
 : `true` or `false`.
 If `true`, mirrors will only be used during pulling if the image reference includes a digest.
+Note that if this is `true`, images referenced by a tag will only use the primary
+registry, failing if that registry is not accessible.
+
+`digest-only`: `true` or `false`.
+: for each mirror in the `mirror` array, the mirror has `digest-only` set to `true` will only be used during pulling if the image reference includes a digest.
```

Comment:
```suggestion
: if `true` an individual mirror will only be used during pulling if the image reference includes a digest.
```

Reply:
The text below is a bit hard to read.  I'd keep it short with: "Note that `digest-only` will be overridden if `mirror-by-digest` is set for the entire registry."